tests/codeception/tests/_bootstrap.php

```diff
@@ -0,0 +1,8 @@
+<?php
+// This is global bootstrap for autoloading
```

Comment:
@sumeetsarna Remove commented code

Reply:
Thanks @juhi123.  I've made the changes. Please check.